### PR TITLE
change model Ourteam -> OurTeam in admin.py

### PR DESCRIPTION
**Pull request by JiwonHwang**
in admin.py
change model name Ourteam to OurTeam

### CODE REVIEW

**Comment by jinpark** [comment]
image

I'm getting this in your code

**Comment by hassanabidpk** [comment]
Any progress on this ?

**Comment by RachellCalhoun** [comment]
@JiwonHwang is handling it I think. If she can't then I will.

**Comment by JiwonHwang** [comment]
I just saw this message. Will work on it

**Comment by hassanabidpk** [comment]
@JiwonHwang  you already had a put request up. So if you commit new change. It will just update the current pull request rather than creating a new PR. :+1:

**Comment by JiwonHwang** [comment]
I made some mistakes at admin.py! 
Working on it now.

File: content/admin.py [hunk 1]
```diff
@@ -7,6 +7,15 @@ class EduProgramAdmin(admin.ModelAdmin):
     list_display = ('title','category', 'order')
 
 
-admin.site.register(OurStory)
-admin.site.register(OurTeam)
+class OurStoryAdmin(admin.ModelAdmin):
+    fields = ['id','order','text']
```
Comment (hassanabidpk) [inline content/admin.py]: Why **id** ❓

File: content/admin.py [hunk 2]
```diff
@@ -7,6 +7,15 @@ class EduProgramAdmin(admin.ModelAdmin):
...
+    list_display=('id', 'order', 'text')
+
+
+class OurTeamAdmin(admin.ModelAdmin):
+    fields = ['id', 'text']
```
Comment (hassanabidpk) [inline content/admin.py]: same for `id` as above

Comment (JiwonHwang) [inline content/admin.py]: @hassanabidpk  I probably have copied from the documentation. I will delete both.

**Comment by JiwonHwang** [comment]
Please check my PR. 
There was a conflict due to changes in admin.py

**Comment by JiwonHwang** [comment]
If no one merges it, my code will keep having conflicts bc of admin.py.
So please check whether other codes are fine, and tell me when to fix the conflict in admin.py.

**Pull request merged**